tmuxp/workspacebuilder.py

```diff
@@ -137,9 +137,15 @@ def build(self, session=None):
                     'Session name %s is already running.' % self.sconf['session_name']
                 )
             else:
-                session = self.server.new_session(
-                    session_name=self.sconf['session_name']
-                )
+                if 'start_directory' in self.sconf:
+                    session = self.server.new_session(
+                        session_name=self.sconf['session_name'],
+                        start_directory=self.sconf['start_directory'],
+                    )
```

Comment:
It's late so I don't want to dredge this up, do we want/need a test for this?

Reply:
I'm happy merging without a test, I can write one later if we need it

Reply:
No worries. It would probably be a good idea to write a test just so we have it